tests/scripts/sync_from_async.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Since the synchronous client delegates all functionality
+# to the asynchronous client, we generate the tests for the
+# sychronous client from the asynchronous ones.
+#
+# Why bother with synchronous tests at all then? Because there
+# is still room for error (typos for example) when delegating.
+# Synchronous tests will catch that.
+# 
+# Yes, this has happened before.
+# 
+# Usage: ./sync_from_async.sh
+
+function async_to_sync() {
+    async_file=$1
+    sync_file=$2
+
+    cp $async_file $sync_file
+
+    sed -i -e 's/async //g' $sync_file
+    sed -i -e 's/await //g' $sync_file
+    sed -i -e 's/momento.aio./momento./g' $sync_file
+    sed -i -e 's/momento.incubating.aio.simple/momento.incubating.simple/g' $sync_file
+    sed -i -e 's/client_async/client/g' $sync_file
```

Comment:
this seems like it could be fraglie, but we can give it a try.

Reply:
Agree. I timeboxed how to write a single set of tests that could be re-used between the two clients and be readable. This was a stop gap until we can find a more graceful approach.